WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Service/BlogDashboardService.swift

```diff
@@ -85,4 +87,18 @@ private extension BlogDashboardService {
         decoder.keyDecodingStrategy = .convertFromSnakeCase
         return try? decoder.decode(BlogDashboardRemoteEntity.self, from: data)
     }
+
+    func localCardsAndGhostCards() -> DashboardSnapshot {
+        var snapshot = parse([:], cards: BlogDashboardRemoteEntity())
```

Comment:
What do you think about leaving a comment here that this creates a snapshot of just the local cards?

Reply:
I extracted it to its own method in 6ba28832e249ff8cd09e96726c8bbade87d690d8